Makefile

```diff
@@ -69,9 +72,11 @@ format: format-js format-python
 format-js:
 	@yarn eslint --ext js --fix "**/*.{js,ts}"
 	@yarn prettier --write "**/*.{js,ts,mjs,cjs,mts,cts,md,sql,json,yml,html,css}"
+
+# https://docs.astral.sh/ruff/formatter/#sorting-imports
 format-python:
-	@python3 -m isort ./
-	@python3 -m black ./
+	@python3 -m ruff check --select I --fix ./
```

Comment:
I think we might just be able to run `ruff check --fix`, since we have the `I` error code set in the config already. However, I am not sure.

Reply:
Changed this in the latest commit.